src/api/mailbox.rs

```diff
@@ -592,3 +608,56 @@ fn receive(
         }
     })
 }
+
+//% lunatic::message::push_tcp_stream(stream_id: u64) -> u64
```

Comment:
Change tcp to udp.  There are a few other places too.

Reply:
Oh wow. I didn't see this before. Thanks for the heads up!
👍